### ISSUE TITLE
Cannot execute QTUM binaries on Raspberry Pi Zero W

### ISSUE THREAD

**Issue by luzzif** [qtumproject/qtum#673]
As the title says, I cannot start `qtumd` (or any other QTUM binary for that matter) on a Raspberry Pi Zero W.

The only output I ever get is `Illegal instruction`.

Could this be an issue with the target ARM version for the QTUM binary being higher than the actual Raspberry Pi Zero W one?

**Comment by luzzif** [comment]
I can tell you the Raspberry Pi Zero W has an `armv6l` CPU.

**Comment by oyyq99999** [comment]
Thanks for reporting. We'll look into it.
BTW what version of Qtum are you running? Since the released binary is supporting armv7+.
Maybe you can try to build it on your Rasp.

**Comment by luzzif** [comment]
I already tried compiling the source code directly on the Raspberry, but it just takes way too long due to the limited memory and swapping madness.

In the end I just gave it up and bought a Pi 3, and it works like a charm.

BTW, I'm not exactly a GCC pro, but isn't it possible to specify a target architecture when building an executable so that I could compile the binary on my home PC and then transfer it to the Raspi through SCP?

I would definitely consider shipping an ARMv6 binary if possible, so that a QTUM full node could be run on a Pi Zero W, which requires less energy compared to a Pi 3.
